hbase-shaded/hbase-shaded-testing-util/pom.xml

```diff
@@ -133,9 +133,9 @@
         </dependency>
         <dependency>
             <groupId>org.codehaus.jackson</groupId>
-            <artifactId>jackson-mapper-asl</artifactId>
+            <artifactId>jackson-jaxrs</artifactId>
             <version>1.9.13</version>
-            <scope>test</scope>
+            <scope>compile</scope>
```

Comment:
the jackson-mapper-asl dependency in the `hbase-shaded-testing-util-tester` module should be removed now I think?